assets/app/scripts/controllers/newfromtemplate.js

```diff
@@ -181,4 +176,17 @@ angular.module('openshiftConsole')
         Navigate.toErrorPage("Cannot create from template: the specified template could not be retrieved.");
       }
     );
+
+    function getFailureObjectName(failure) {
+      if (!failure.data || !failure.data.details) {
+        return null;
+      }
+
+      var details = failure.data.details;
+      if (details.kind) {
+        return (details.id) ? details.kind + " " + details.id : details.kind;
+      }
+
+      return details.id;
```

Comment:
@liggitt No, but we handle when this method returns null or undefined above (line 118).